internal/test/t2/t2_test.go

```diff
@@ -0,0 +1,24 @@
+package t2
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestEmptyKeyword(t *testing.T) {
+	var buf = new(bytes.Buffer)
+	cmd := exec.Command("gocc", "t2.bnf")
+	cmd.Stdout = buf
+	err := cmd.Run()
+
+	e, ok := err.(*exec.ExitError)
+	if !ok || e.Success() {
+		t.Fatalf("gocc t2.bnf should return with exit error, but get %v.", err)
+	}
+	expectedStderr := `maybe you missing the "empty" keyword in "B : \t<<  >>"`
```

Comment:
Hmmm, seems tests are passing, but this string is incorrect, are we sure this test is being run?

Reply:
no, test is correct, test check `!strings.Contains(actual, expectedStderr)` not `actual == expectedStderr`

Reply:
and add log `t.Logf("actual:\t%s\nexpectedStderr:\t%s\n", actual, expectedStderr)`, print:
```
	t2_test.go:22: actual:	Parse error: Error: $(0)  @ 10:2 maybe you missing the "empty" keyword in "B : \t<<  >>"
		
		
		expectedStderr:	maybe you missing the "empty" keyword in "B : \t<<  >>"
```

Reply:
I thought actual was now supposed to be `Maybe you are missing the \"empty\" keyword`